mediator/med_phases_ocnalb_mod.F90

```diff
@@ -102,6 +97,8 @@ subroutine med_phases_ocnalb_init(gcomp, ocnalb, rc)
     real(R8), pointer        :: ownedElemCoords(:)
```

Comment:
since all pointers init to null(), it would be nice to init this one with null().

Reply:
Good point. I'm going to go through and set all pointers to null for consistency.

Reply:
I just pushed back all local pointers being set to null(). That is what Denise is testing with - and I will redo the tests tonight.